addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -99,7 +99,7 @@ export class MassMailingHtmlField extends HtmlField {
         const $editable = this.wysiwyg.getEditable();
         const initialHtml = $editable.html();
         await this.wysiwyg.cleanForSave();
-        await this.wysiwyg.saveModifiedImages(this.$content);
+        await this.wysiwyg.saveModifiedImages();
```

Comment:
Is this changed intended ? It doesn't look to be linked to the issue at hand and, if I'm not mistaken, is not accounted for in the commit message. Did I miss something ?

Reply:
Ah I guess I didn't explain that clearly in the commit. ``this.$content`` is only set in the old field_html, not html_field. That's the reason for the original traceback.

It seems to be defaulting to the right value now in the parent class.

Reply:
If you could update the commit message to include this important piece of information that would be awesome, thanks ! :)

Reply:
This was fixed as a side-effect shortly after, so it doesn't pose any issue right now.

This diff is really just a formality, I can still make it into its own commit for sure though :)